src/CallbackStream.php

```diff
@@ -126,7 +122,7 @@ public function isWritable(): bool
     /**
      * {@inheritdoc}
      */
-    public function write($string): void
+    public function write(string $string): int
```

Comment:
Will `v1` still work, with these types applied? :thinking:

Reply:
Yes: https://github.com/laminas/laminas-diactoros/actions/runs/4629754113/jobs/8190430823?pr=136#step:3:387 (that demonstrates installation of 1.1 for a successful PHPUnit run).